style/style.css

```diff
@@ -357,9 +357,9 @@ header {
 .team__members {
   display: grid;
   grid-template-columns: repeat(3, 1fr);
-  grid-template-rows: repeat(2, 1fr);
```

Comment:
You don't necessary need grid for this. Grid is used when you have multiple rows and columns, while here you only have one row. You can change the display to flex and add gap.